lib/tasks/export.rake

```diff
@@ -0,0 +1,19 @@
+require 'public_organization_exporter'
+require 'fileutils'
+
+namespace :export do
+  desc "Exports profiles to public/export/public_organizations.csv,
+        public/export/public_organizations.xls and public/export/public_organizations.json"
+  task profiles: :environment do
```

Comment:
Rake task name could be better "organizations" instead of profiles :) I know I know... copy & paste sometimes its hard to doublecheck 😅

Reply:
:sob: